### PR TITLE
Created User Entity and dtos

### PR DESCRIPTION
**Pull request by fullstack-Maestro**

### CODE REVIEW

File: src/OneFit.Service/DTOs/UserViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿namespace OneFit.Service.DTOs;
+
+public class UserViewModel
+{
+    public long Id { get; set; }
+    public string FirstName { get; set; }
+    public string LastName { get; set; }
+    public string Phone { get; set; }
+    public string Password { get; set; }
```
Comment (makhammadsoliyev) [inline src/OneFit.Service/DTOs/UserViewModel.cs]: Password must not be shown

**Review by makhammadsoliyev — CHANGES_REQUESTED** [review]
UserViewModel

File: src/OneFit.Service/DTOs/UserViewModel.cs (hunk 1 above)
Comment (fullstack-Maestro) [reply src/OneFit.Service/DTOs/UserViewModel.cs:9]: Can you clarify your comment

Comment (fullstack-Maestro) [reply src/OneFit.Service/DTOs/UserViewModel.cs:9]: Did you mean Hashing

**Review by Muhammadjonov-D — APPROVED** [review]

**Review by makhammadsoliyev — COMMENTED** [review]
Good

**Review by makhammadsoliyev — APPROVED** [review]

**Pull request merged**